crates/wysiwyg/src/dom/dom_handle.rs

```diff
@@ -112,3 +114,33 @@ impl DomHandle {
         Self::from_raw(path)
     }
 }
+
+impl PartialOrd<Self> for DomHandle {
+    fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+impl Ord for DomHandle {
+    fn cmp(&self, other: &Self) -> Ordering {
```

Comment:
Is this equivalent to `self.raw.cmp(other.raw)`? I assume this exists and is a lexical compare?

Reply:
I'd have to double check. I implemented this myself because I didn't trust the `Vec::cmp` default implementation to work as we expected, but if it does it would simplify everything.